tools/mfassignr/mfassignr_mfassignCHO.xml

```diff
@@ -22,6 +22,19 @@
         iso <- read.delim("$input_iso", sep="\t")
         #end if
         SN = $sn_ratio * $kmdn
+        POEx = 0
+        NOEx = 0
+
+        #if "$assign_typeof_ions"=="yes"
+            #if $ionMode=="pos"
+            POEx = 1
+            NOEx = 0
+            #end if
+            #if $ionMode=="neg"
+            POEx = 0
+            NOEx = 1
+            #end if
```

Comment:
same as above. This could even be shared as a token in the macro, theoretically.